stablehlo/tests/vhlo/stablehlo_legalize_to_vhlo.mlir

```diff
@@ -978,6 +1000,18 @@ func.func @op_all_to_all(%arg0: tensor<4x16xf32>) -> tensor<16x4xf32> {
   func.return %0 : tensor<16x4xf32>
 }
 
+// CHECK-LABEL: "op_all_to_all_variadic"
```

Comment:
Let's make this consistent and have a "default" test here too?

Reply:
Done